dashel/dashel-common.cpp

```diff
@@ -45,11 +45,11 @@
 #include <ostream>
 #include <sstream>
 #ifndef _WIN32
-	#include <netdb.h>
-	#include <sys/socket.h>
-	#include <arpa/inet.h>
+#include <netdb.h>
```

Comment:
I like this indentation, it makes the #ifdefed code more readable.

Reply:
I agree on principle, but that's not something clang format can deal with as of 5.0.
As of 6.0 it can indent that properly but I didn't want to rely on an not-yet-released version.

Reply:
I guess we could but we would have to every single time.
I prefer imperfect but automated that the other way around, don't you agree ?

Btw clang 6 does something like : 
```
#ifdef XX
#    include "foo"
#endif
``` 

 - See `IndentPPDirectives` in https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
> I prefer imperfect but automated that the other way around, don't you agree ?

The tool should help to reach our goal, not the contrary. It is fine to do things manually if the result is readable and the additional load not too large. I guess the tool won't be launched too often.

The clang 6 version is fine for me.